apps/web/src/app/[orgShortCode]/convo/[convoId]/_components/reply-box.tsx

```diff
@@ -109,9 +129,33 @@ export function ReplyBox({ convoId }: { convoId: TypeId<'convos'> }) {
                       {emailIdentities?.emailIdentities.map((email) => (
                         <SelectItem
                           key={email.publicId}
-                          value={email.publicId}>
-                          <span>
-                            {email.username}@{email.domainName}
+                          value={email.publicId}
+                          className="[&>span:last-child]:w-full">
+                          <span className="flex items-center justify-between">
+                            <span
+                              className={cn(
+                                !email.usageAllowed && 'text-muted-foreground'
+                              )}>
+                              {`${email.sendName} (${email.username}@${email.domainName})`}
```

Comment:
just use `disabled: usageAllowed` on the `SelectItem` instead of manually setting the styles

Reply:
That will disable the tooltip too
I tried it